keras_nlp/src/utils/transformers/convert_mistral_test.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+from keras_nlp.src.models.mistral.mistral_causal_lm import MistralCausalLM
+from keras_nlp.src.tests.test_case import TestCase
+
+
+class TestTask(TestCase):
+    @pytest.mark.large
+    def test_convert_tiny_preset(self):
+        model = MistralCausalLM.from_preset("hf://mistralai/Mistral-7B-v0.1")
```

Comment:
This is too big to run in our automated testing regularly. @ariG23498 can you detail what you did to make `hf://ariG23498/tiny-gemma-test`?

Reply:
[Here](https://colab.research.google.com/gist/ariG23498/e1b55659c3cfbdc3266c58497b3a4814/generic-model-build.ipynb) is a detailed code to build a small test model and how to upload that to hub.

@cosmo3769 could you take a look at it?

Reply:
@ariG23498 Sure.